packages/chrome/src/styled/subnav/StyledSubNavItemText.ts

```diff
@@ -20,12 +21,26 @@ export interface IStyledSubNavItemTextProps {
   isWrapped?: boolean;
 }
 
-export const StyledSubNavItemText = styled.span.attrs<IStyledSubNavItemTextProps>(props => ({
+const sizeStyles = (props: ThemeProps<DefaultTheme>) => {
+  const baseLineHeight = props.theme.space.base * 5;
+  const verticalMargin = math(`(${getSubNavItemHeight(props)} - ${baseLineHeight}) / 2`);
```

Comment:
I suppose a `stripUnit` could be used on `getSubNavItemHeight` if you're worried about the performance for `math`.

Reply:
I'm not too worried about this area compared to others. We can tweak if necessary.